cockroachcloud/export-metrics.md

```diff
@@ -233,6 +239,12 @@ Where:
 
 To update an existing {{ site.data.products.dedicated }} metrics export configuration, make any necessary changes to your cloud provider configuration (e.g., AWS CloudWatch or Datadog), then issue the same `POST` Cloud API command as shown in the [Enable metrics export](#enable-metrics-export) instructions for your cloud provider with the desired updated configuration. Follow the [Monitor the status of a metrics export configuration](#monitor-the-status-of-a-metrics-export-configuration) instructions to ensure the update completes successfully.
 
+## Disable Cloud Console's local storage of metrics (optional)
+
+If you rely on external tools such as AWS CloudWatch or Datadog for storing and visualizing your cluster's time-series metrics, Cockroach Labs recommends that you disable the Cloud Console's storage of time-series metrics by setting the [`timeseries.storage_enabled` cluster setting](/docs/v23.1/cluster-settings.html#setting-timeseries-storage-enabled) to `false`.
```

Comment:
We should also call out that the DB Console is disabled for Dedicated clusters. 

And @thtruo, can you confirm that the Cloud Console uses the time-series data store as well for Dedicated? If i recall correctly, it uses Prometheus?

Reply:
@kevin-v-ngo you're correct - Cloud Console uses Prometheus metrics, a different source from the DB Console

@mdlinville we should should update the copy to just say "disable the DB Console's storage of time-series metrics"

Reply:
I'll revert the addition of this section in the Cloud docs, and mention that DB Console is disabled for Cloud clusters.